backend/package.json

```diff
@@ -18,6 +18,7 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
+    "@aws-sdk/client-sns": "^3.7.0",
     "@sentry/node": "^5.16.1",
     "async-retry": "^1.3.1",
     "aws-sdk": "^2.814.0",
```

Comment:
Since you're using the v3 SDK in this change, are there any plans to migrate from v2 to v3 entirely in the future?

Reply:
Good point. Guess we can open a new issue to track the effort required to move to the new V3 sdk.